Task2/content.css

```diff
@@ -0,0 +1,23 @@
+.content div {
+    margin: 10px;
+    background: rgb(237,238,240);
+    display: inline-block;
+}
+.list a {
+    text-decoration-style: dashed;
+    margin: 3px;
+    padding: 2px;
+}
```

Comment:
Dashed underline looks really ugly )))
How make it read [here](http://htmlbook.ru/faq/kak-dobavit-punktirnoe-podcherkivanie-k-ssylkam)